docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -26,11 +26,12 @@ Kyma comes with a local wildcard self-signed [certificate](../../../installation
 
 Follow these steps to "always trust" the Kyma certificate on macOS:
 
-1. Open the Keychain Access application. Go to **File**, then select **Import items...** and import the Kyma [certificate](../../../installation/certs/workspace/raw/server.crt).
-2. Go to the **Certificates** view and find the `*.kyma.local` certificate you imported.
-3. Righ-click on the certificate and select **Get Info**.
-4. Expand the **Trust** list and set **When using this certificate** to **Always trust**.
-5. Close the certificate information window and enter your system password to confirm the changes.
+1. Open the Keychain Access application. Select **System** from the **Keychains** menu.
+2. Go to **File**, then select **Import items...** and import the Kyma [certificate](../../../installation/certs/workspace/raw/server.crt).
+3. Go to the **Certificates** view and find the `*.kyma.local` certificate you imported.
+4. Right-click on the certificate and select **Get Info**.
```

Comment:
If we have "Click sth" in the guidelines, I guess that applies to "Right-click" as well.